City-of-Helsinki/drupal-helfi-platform-config
# UHF-4462 update sitemap module

# Sitemap [UHF-4462](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4462)
Sitemap didn't work and required some configurating. In the end every instance must be configured by hand.

## What was done
- Bumped up major version for simple_sitemap
- Small fix to helfi_proxy module to fix the sitemap page

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `make fresh`
* Make sure helfi_proxy-module is updated to 2.1.2 or higher 
    * `composer update drupal/helfi_proxy`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-4462_update_sitemap_module -W`
* Run `make drush-updb drush-cr`

## How to test
* Login as admin and rebuild the sitemap (rebuild queue & generate)
   * ´/admin/config/search/simplesitemap´

* Go and see that the sitemap is generated properly
   * ´/sitemap.xml´

## Other PRs
* proxy fix (released) https://github.com/City-of-Helsinki/drupal-module-helfi-proxy/pull/35
